index.js

```diff
@@ -116,6 +122,29 @@ function compileTS(tsconfigFile, projectPath, tsinclude) {
   });
 }
 
+function mungeCompilerOptions(compilerOptions) {
+  let optionDefaults = {
+    lib: []
+  };
+  let optionOverrides = {
+    target: 'es2017',
+    module: 'es2015',
+    outDir: 'dist'
+  };
+
+  compilerOptions = Object.assign({}, optionDefaults, compilerOptions, optionOverrides);
```

Comment:
The more I think about it, the less happy I am with this, including the `lib: []`. Defining `lib` as `[]` can end up overriding the typescript default that is applied when you leave `lib` undefined.